Tests/BrowserServicesKitTests/Resources/remote-messaging-config.json

```diff
@@ -237,9 +237,6 @@
     {
       "id": 8,
       "attributes": {
-        "isNetPWaitlistUser": {
-          "value": true
-        },
```

Comment:
@samsymons - I'm removing this and changing the test that checks this exists.  Is this fine?  Is there anything else we need to do here to adjust things?  Do we need to change anything in remote?

Reply:
Nope, all good :+1: